.stickler.yml

```diff
@@ -0,0 +1,7 @@
+---
+linters:
+    eslint:
+        config: '.eslintrc.json'
+files:
+  ignore:
+      - '.eslintrc.json'
```

Comment:
Please do not ignore the config file. Use this instead: https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml


Is there a reason why you ignoring this config file?